modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonLegacyClientCodegen.java

```diff
@@ -333,11 +334,45 @@ public void postProcessPattern(String pattern, Map<String, Object> vendorExtensi
                 }
             }
 
-            vendorExtensions.put("x-regex", regex);
             vendorExtensions.put("x-modifiers", modifiers);
         }
     }
 
+    /**
+     * // CS427 Issue link: https://github.com/OpenAPITools/openapi-generator/issues/6675
+     *
+     * Fixes errors caused by adding backslash to some regex patterns
+     *
+     * @parem pattern Regex pattern defined in the OpenAPI specification file
+     * @parem vendorExtensions HashMap which stores strings used in the generated code templates
+     * @return resultPattern Regex pattern which is or is not transformed
+     */
+    public String fixRegexBackSlashError(String pattern, Map<String, Object> vendorExtensions) {
```

Comment:
How about setVendorExtensionRegex?
Yes it fixes a bug but more importantly it sets the vendor extension regex